projects/igniteui-angular/src/lib/simple-combo/simple-combo.component.ts

```diff
@@ -240,7 +240,9 @@ export class IgxSimpleComboComponent extends IgxComboBaseDirective implements Co
                 this.filterValue = this.searchValue = this.comboInput.value;
                 return;
             }
-            this.filterValue = this.searchValue = '';
+            if (!this.comboInput.value) {
```

Comment:
now if `this.comboInput.Value` is set, e.g. combo has initial value, same issue is reproducible.